remind.py

```diff
@@ -372,9 +372,18 @@ def _rem_clean(rem):
         """Transform newlines and '[' in string."""
         return rem.strip().replace('\n', '%_').replace('[', '["["]')
 
-    def to_remind(self, vevent, label=None, tail=None, sep=" ", delta=None,
-                  tdelta=None, trepeat=None, priority=None, tags=None):
-        """Generate a Remind command from the given vevent"""
+    def to_remind(self, vevent, **kwargs):
```

Comment:
This changes the API quite a bit, also, using strings to access the dict tricks some static analysis tools. I would propose to go back to the old way.